Source/Core/Core/HW/EXI/BBA/BuiltIn.cpp

```diff
@@ -727,6 +779,22 @@ sf::Socket::Status BbaTcpSocket::Connect(const sf::IpAddress& dest, u16 port, u3
   return this->connect(dest, port);
 }
 
+sf::Socket::Status BbaTcpSocket::GetPeerName(sockaddr_in* addr)
```

Comment:
This isn't a lot of code...but maybe add a helper function that both implementations call?